apps/agent/package.json

```diff
@@ -8,9 +8,11 @@
     "build": "mastra build",
     "lint": "eslint . --cache --ext .js,.jsx,.ts,.tsx",
     "lint:fix": "eslint . --fix --cache --ext .js,.jsx,.ts,.tsx",
-    "db:generate": "drizzle-kit generate",
-    "db:push": "drizzle-kit push:pg",
-    "db:studio": "drizzle-kit studio"
+    "db:generate": "prisma generate",
+    "db:push": "prisma db push",
+    "db:studio": "prisma studio",
+    "db:migrate:dev": "prisma migrate dev",
+    "db:migrate:deploy": "prisma migrate deploy"
```

Comment:
Should we start the migration step during the build on the remote server? I don’t see any updates in the Dockerfile